Update wca-states.md to show one list without descriptions

Mollerz opened the pull request:
Current ongoing discussion - seems like the route we are headed is to keep the list of countries the same, however remove the descriptions and reasons for allowing certain countries that are not UN Member/Observer States to avoid any political bias or misinformation.

Mollerz committed:
Update wca-states.md to show one list without descriptions

Current ongoing discussion - seems like the route we are headed is to keep the list of countries the same, however remove the descriptions and reasons for allowing certain countries that are not UN Member/Observer States.

jfly commented on wca-states.md:
```diff
@@ -3,10 +3,7 @@
-The WCA recognizes a total of 199 independent countries.
-
-
-## <states-list>UN Member States:
```
The build is failing (https://travis-ci.org/thewca/wca-regulations/builds/418300471#L474), presumably because you removed the `## <states-list>` token. Maybe try changing line 6 to:

```
<states-list> The WCA recognizes a total of 199 countries.
```

?

Mollerz replied:
Yeah I don't know how coding works :P I will add that back in, it makes sense!

jfly replied:
Ok, sorry, nevermind! It looks like the compiler expects you to include the `##` as well. (relevant code here: https://github.com/thewca/wca-regulations-compiler/blob/6b6aee02de716bb2c7a1b1626200b61bcc364bee/wrc/parse/lexer.py#L77). I think to get the build passing, you'll have to change this to:

```
## <states-list> The WCA recognizes a total of 199 countries.
```

(and I'd consider changing that period at the end to a colon)

It's also worth noting that adding the `##` will change how this looks over on https://www.worldcubeassociation.org/regulations/countries/ (the sentence will look like a heading), which may be undesirable.

@viroulep because he's the expert on this.

jfly commented:
Thanks for taking care of this, @Mollerz!

Mollerz committed:
Update wca-states.md

Mollerz committed:
Update wca-states.md

I'm hoping this works, we have the title and formatting but does not have any text following it as we don't actually need a header anymore.

Mollerz commented:
We should also have a private page that probably board/WRC can access so that internally we understand the reasoning for having this list the way it is. In the future when new countries are born, or if there are any changes we need to make, people can easily see the reasoning behind our current list instead of having to reason it from scratch themselves.

jfly commented:
James, it looks like the build is still failing, perhaps due to some trailing whitespace?

I think it would be great for us to have an internal document for this, I think a Google doc would be the right place for this for now.

Mollerz commented:
I will create the Google Doc - I think waiting for @viroulep might be best to see what we can do about it failing :)

lgarron commented on wca-states.md:
```diff
@@ -1,13 +1,13 @@
 <version>Version: June 19, 2017
+  
+  
+The WCA recognizes a total of 199 states/regions:
```
I would prefer to avoid "region", since we already use "region" to refer to explicitly refer to nations, continents, and the world: https://www.worldcubeassociation.org/regulations/#9i1

Could we just say "states?

Baiqiang replied:
@lgarron If we used only one word, Chinese people would treat China and Taiwan as the same kind/level of a state (or some word else). This is like the current word "country".

viroulep replied:
@lgarron please check the current email thread about this matter, were this specific wording is being discussed (we'd rather have the discussion at one single place).

lgarron commented on wca-states.md:
```diff
@@ -1,13 +1,13 @@
 # <wca-title>WCA List of Recognized Countries
```
Should the title be updated?

viroulep commented:
Indeed the build is simply failing because of trailing whitespaces on lines 4, 5, and 10.

Mollerz committed:
Update wca-states.md

Removed trailing whitespace, updated to countries/regions per the email chain.

Mollerz committed:
Update wca-states.md

jfly commented on wca-states.md:
```diff
@@ -3,10 +3,10 @@
 
 
-## <states-list>UN Member States:
+## <states-list>
```
Looks like the build is still failing, unfortunately. I think that's because the regex here: https://github.com/thewca/wca-regulations-compiler/blob/6b6aee02de716bb2c7a1b1626200b61bcc364bee/wrc/parse/lexer.py#L77 is expecting there to be at least one character after `<states-list>`. I think there are two ways we can fix this:

1. Change the wca-regulations-compiler to not require anything after `<states-list>`. If we're going to that effort, then it probably doesn't make sense for it to require the `##` anymore, either.
2. Combine lines 6 and 9 so this looks like:

```
## <states-list> The WCA recognizes a total of 199 countries/regions:
```

Mollerz replied:
That makes sense - we don't have to format it with ## either right? We don't want to make this a huge title text after all.

jfly replied:
@viroulep, do you see any issues with changing the regs compiler as I've described in 1) above?

viroulep replied:
Fixed in thewca/wca-regulations-compiler@3ebbd8fe8fec7c9167296f59b2677696fe61a954.

cubewhiz commented:
This change has been approved by the Board, so when the code is ready, you can proceed with this change.

viroulep commented:
I just noticed this PR has been created against the `draft` branch, which is not what we want in this case (see [here](https://github.com/thewca/wca-regulations/wiki/On-submitting-pull-requests) why) as we want it to land in `official` without going through the Regulations release process.

I'll see what I can do.

viroulep commented:
Superseded by #621.

The commits were on top of all the draft changes that we don't want in official yet, so what I did is the following:
  - squashed James commits into one clean commit
  - cherry-picked this commit on top of official, and submitted #621

The diff is the same, so I'll go ahead and publish this!
